sdcm/scylla_bench_thread.py

```diff
@@ -124,6 +124,8 @@ def verify_results(self):
         return sb_summary, errors
 
     def _run_stress_bench(self, node, loader_idx, stress_cmd, node_list):
+        stress_cmd = re.sub(r"SCT_TIME", f"{int(time.time())-480}", stress_cmd)
```

Comment:
Add a comment explaining what is this "480" and why it's needed.

Reply:
https://github.com/scylladb/scylla-cluster-tests/pull/3318